doc/ref/plotting_options/size_layout.ipynb

```diff
@@ -0,0 +1,366 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2ae44fa2-2ef0-4204-b9bb-b6051dfefd7c",
+   "metadata": {},
+   "source": [
+    "# Size And Layout Options\n",
+    "\n",
+    "\n",
+    "```{eval-rst}\n",
+    ".. plotting-options-table:: Size And Layout Options\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c20d146d-c67f-40be-b505-edadc4e94bd7",
+   "metadata": {},
+   "source": [
+    "(option-aspect)=\n",
+    "## `aspect`\n",
+    "\n",
+    "The `aspect` option controls the **frame** shape by setting the ratio of its width to its height. By default (`aspect=None`), HoloViews chooses an appropriate ratio automatically. Use:\n",
```

Comment:
To understand, what do you mean with HoloViews choosing an appropriate ratio automatically?

Reply:
It means the ratio used is either the default or the specified frame width/height. I think I'll have to rewrite that as it is a bit vague.